content/learn/overview.md

```diff
@@ -20,36 +26,129 @@ layout: "overview"
-                    <br>Read <a href="/download/addons/">here</a> to learn how to install them.</p>
+<p><b>GRASS GIS</b> runs on Linux, Mac and Windows. It comes with 
+both an intuitive graphical user interface (GUI) and a command line 
+syntax useful for automation and production.</p>
+<p><b>GRASS GIS</b> contains over 
+<b><a href="https://grass.osgeo.org/grass-devel/manuals/full_index.html" target="_blank">500 modules</a></b> 
+to process and render geographic data. The software allows to manipulate
+a variety of raster, vector and 3D formats, and run simple to advanced 
+spatial analysis and modeling. GRASS GIS can connect to spatial databases 
+but also interface with a variety of third-party systems and libraries.</p>
+</div>
 
-  </div>
+</div>
+</div>
+
+
+<div class="container">
+<div class="row mt-20">
+
+Some selected relevant features are:
+
+<ul>
+<img class="bsh" src="/images/gallery/temporal/summer_solstice_centennial.gif" width="40%" style="float:right;padding-left:10px">
+<li>Support for nearly all common GIS file formats through GDAL-OGR library.</li>
+<li>Raster analysis: map algebra, interpolation, masking, landscape analysis.</li>
+<li>3D raster (voxel) analysis: 3D map algebra, 3D interpolation, 3D visualization.</li>
+<li>SQL: database interfaces to PostgreSQL, SQLite, etc.</li>
+<li>Geo-coding of raster and vector maps.</li>
+<li>Point cloud analysis: LiDAR, interpolation.</li>
+<li>Terrain analysis: contour and surface generation, cost-path and slope-aspect analysis, hydrological tools.</li>
+<li>Vector analysis: topology corrections and generalization, buffer, overlays, network analysis.</li>
+<li>Temporal framework: support for raster and vector time series management and analysis, <strong>big spatio-temporal environmental data</strong>.</li>
+<li>Image processing: aerial, UAV and satellite data (MODIS, Landsat, Sentinel, etc.), supervised and unsupervised classification, full <strong>Object Based Image Analysis</strong> (OBIA) chain.</li>
+<li>Spatial statistics: correlation and covariance analysis, regression, zonal statistics.</li>
+</ul>
 
 </div>
 </div>
 
+### Interfaces
+
+<div class="container">
+<div class="row mt-20">
+
+<div class="col-lg-6 text-center">
+<img class="bsh" src="/images/other/grass_through_qgis.png" width="80%" alt="GRASS GIS through QGIS">
 </div>
+
+<div class="col-lg-6">
+GRASS GIS can be used through different interfaces: 
+<ul>
+<li>The simplest for new users is the <strong>Graphical User Interface</strong> (GUI).</li>
+<li>Power users use the text-based <strong>command line interface</strong> (CLI).</li>
+<li><strong>C API</strong> for programming.</li>
+<li>Scripting library and object-oriented <strong>Python API</strong>.</li>
+<li><strong>Web interface</strong> through WPS servers.</li>
+<li>QGIS provides two different ways to run GRASS GIS modules.</li>
+<li>R provides an interface to GRASS GIS through the package <strong><i>rgrass7</i></strong>.</li>
+</ul>
```

Comment:
- maybe link to the respective (Wiki) pages?
- add link to https://staging.grass.osgeo.org/learn/tryonline/ ?

Reply:
yes, on it :)